src/test/java/uk/gov/pay/adminusers/resources/ServiceResourceCreateIT.java

```diff
@@ -1,37 +1,117 @@
+                .body("default_billing_address_country", is("GB"))
+                .body("agent_initiated_moto_enabled", is(false))
+                .body("takes_payments_over_phone", is(false))
+                .body("experimental_features_enabled", is(false))
+                .body("internal", is(false))
+                .body("archived", is(false))
+                .body("redirect_to_service_immediately_on_terminal_state", is(false))
+                .body("collect_billing_address", is(true));
+    }
+
+    @Test
+    void can_create_default_service_with_empty_request_body() {
+        var validatableResponse = givenSetup()
+                .when()
+                .contentType(JSON)
+                .post("v1/api/services")
+                .then()
+                .statusCode(201)
+                .body("created_date", is(not(nullValue())))
+                .body("gateway_account_ids", is(emptyIterable()))
+                .body("service_name", hasEntry("en", "System Generated"))
+                .body("name", is("System Generated"));
+        
+        assertStandardFields(validatableResponse);
+    }
+    
+    @Test
+    void can_create_default_service_with_gateway_account_ids_only() {
+        var validatableResponse = givenSetup()
+                .when()
+                .contentType(JSON)
+                .body(Map.of("gateway_account_ids", List.of("1", "2")))
+                .post("v1/api/services")
+                .then()
+                .statusCode(201)
+                .body("created_date", is(not(nullValue())))
+                .body("gateway_account_ids", is(List.of("1", "2")))
+                .body("service_name", hasEntry("en", "System Generated"))
+                .body("name", is("System Generated"));
+
+        assertStandardFields(validatableResponse);
+    }
+
+    @Test
+    void can_create_default_service_with_service_name_only() {
+        var validatableResponse = givenSetup()
+                .when()
+                .contentType(JSON)
+                .body(Map.of("service_name", Map.of(ENGLISH.toString(), "Service name", WELSH.toString(), "Welsh name")))
+                .post("v1/api/services")
+                .then()
+                .statusCode(201)
+                .body("created_date", is(not(nullValue())))
+                .body("gateway_account_ids", is(emptyIterable()))
+                .body("service_name", hasEntry("en", "Service name"))
+                .body("service_name", hasEntry("cy", "Welsh name"))
+                .body("name", is("Service name"));
+
+        assertStandardFields(validatableResponse);
+    }
+    
+    @Test
+    void return_bad_request_when_invalid_supported_language_provided() {
+        givenSetup()
+                .when()
+                .contentType(JSON)
+                .body(Map.of("service_name", Map.of("fr", "Service name")))
+                .post("v1/api/services")
+                .then()
+                .statusCode(400)
```

Comment:
Returning a 422 might be more correct here, but getting Jersey to do this from within a custom deserialiser is very hard to do. We return a [400 in other places](https://github.com/alphagov/pay-connector/blob/master/src/test/java/uk/gov/pay/connector/charge/resource/ChargesApiResourceValidationTest.java#L199) so I think this is acceptable.

Reply:
I agree this is fine for an internal API.